pex/vendor/__main__.py

```diff
@@ -201,6 +194,18 @@ class VendorizeError(Exception):
     """Indicates an error was encountered updating vendored libraries."""
 
 
+def find_site_packages(prefix_dir):
```

Comment:
A comment explaining to the naive reader why we can't use `find_site_packages_dir` here would be valuable.

Reply:
Like this?: https://github.com/pantsbuild/pex/blob/e38fdc5047e7309b3b78d71ee95dfa73f457f471/pex/vendor/__main__.py#L210-L216

Reply:
I guess that is sufficient. Plus if anyone tries to change this, tests will break.